docs/http-api/entry-api.mdx

```diff
@@ -404,6 +404,8 @@ description={
     Since version 1.4, the method has the <code>continuous query</code> flag. If it is true, the current query will not be discarded if there are no records. A client can ask them later. The query will not be removed until its TTL has expired. The <code>stop</code> parameter is ignored for continuous queries.
```

Comment:
the API doc for _Query records for a time interval_ starts to have a long description. Maybe adding bullet points can make it clearer or more compact?